app/jsx/external_apps/components/Header.jsx

```diff
@@ -31,7 +32,11 @@ define([
             <p>{I18n.t('Apps are an easy way to add new features to Canvas. They can be added to individual courses, or to all courses in an account. Once configured, you can link to them through course modules and create assignments for assessment tools.')}</p>
             <p dangerouslySetInnerHTML={{ __html: paragraph }}></p>
           </div>
+
+          <SFUPrivacyNotice alertStyle="alert" usage="external_apps" />
```

Comment:
Oh well. At least the letters "SFU" are in there, so it's not too hard to spot/search for.